services/spark/helm/spark/values.yaml

```diff
@@ -0,0 +1,83 @@
+master:
+  image:
+    repository: "kubelytics/spark"
+    tag: "2.1.2"
+    pullPolicy: "IfNotPresent"
+    command: /etc/config/spark/run.sh
+  replicas: 1
+  servicePort: 7077
+  containerPort: 7077
+  
+  resources: {}
+    # limits:
+      # cpu: 1
+      # memory: "3Gi"
+    # requests:
+      # cpu: 1
+      # memory: "3Gi"
+  
+  webAdmin:
+    servicePort: 8080
+    containerPort: 8080
+
+worker:
+  image:
+    repository: "kubelytics/spark"
+    tag: "2.1.2"
+    pullPolicy: "IfNotPresent"
+    command: /etc/config/spark/run.sh
```

Comment:
Is the command really a value? Why would anyone configure this?

Reply:
So that v3io (or AWS or Azure) charts can bootstrap their containers with the script they wish